src/scripts/main.js

```diff
@@ -1 +1,93 @@
 'use strict';
+
+const notification = document.createElement('div');
```

Comment:
The notification element is shared across all promises. This can lead to conflicts as each promise modifies the same element. Consider creating separate notification elements for each promise to ensure they do not interfere with each other.